CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ This project employs a versioning scheme described in [RELEASE.md](RELEASE.md#ve
 
 ## [Unreleased]
 
+## [1.25.7]
+
+### Fixed
+
+- Fix to call addon on k8sMaintain phase [#638](https://github.com/cybozu-go/cke/pull/638)
+- Add a comment for the added condition whether calling the addon [#640](https://github.com/cybozu-go/cke/pull/640)
```

Comment:
We don't need to add PR #640 here. Let's focus on the user-facing changes

Reply:
Thank you.
I remove the line about #640.